### ISSUE TITLE
Add js methods to mixins component to make it more convenient to work with.

### ISSUE THREAD

**Issue by caseyyee** [aframevr/aframe#2509]
It would be very convenient to have methods to add and remove mixins using using Javascript in the same way class names are handled:

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

**Comment by dmarcos** [comment]
Yes, we could maybe have `el.addMixin()` / `el.removeMixin()`?

**Comment by caseyyee** [comment]
Yeah, that makes more sense rather than digging into the component.  👍

**Comment by alfa256** [comment]
Since order is important, where would `el.addMixin()` add ?
Maybe `el.appendMixin()` and `el.prependMixin()` ?
